pkg/destroy/gcp/gcp.go

```diff
@@ -118,6 +133,11 @@ func (o *ClusterUninstaller) Run() error {
 		return errors.Wrap(err, "failed to destroy cluster")
 	}
 
+	raw, err := json.Marshal(o.pendingItemTracker.removedQuota)
+	if err != nil {
+		return errors.Wrap(err, "failed to marshal quota")
+	}
+	fmt.Println(string(raw))
```

Comment:
Alternatively, it might be worth placing this behind a flag and outputting to an file for consumption.

Reply:
Yep - I was hoping to flag-gate it, but since there's no way to currently pass any run-time parameters down to the Destroyers I was not sure how to best do that.

Reply:
Can we still get rid of this line or switch to logging? Or are you just keeping it around for debugging?